ui/packages/catalog-ui-search/src/main/webapp/component/workspace-item/workspace-item.view.js

```diff
@@ -18,26 +18,68 @@ define([
     'marionette',
     'underscore',
     'jquery',
-    './workspace-item.hbs',
```

Comment:
Any reason we moved away from this define in other places (which I am glad to see) but didn't do it here?